### PR TITLE
VIH-5844 Video_Web___Participants_are_able_to_access_a_closed_hearing_after_30_minutes

### PR DESCRIPTION
**Pull request by RocheL**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-5844
### Change description ###
 - updated participant hearing list to display status closed for closed hearings
**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb.Services.Generator/video-api.nswag [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "defaultVariables": null,
   "documentGenerator": {
     "fromDocument": {
-        "url": "https://vh-video-api-dev.azurewebsites.net/swagger/v1/swagger.json",
```
Comment (npoxon) [inline VideoWeb/VideoWeb.Services.Generator/video-api.nswag:6]: Lionel can you change this back? It better to generate swagger from dev not local

Comment (RocheL) [reply VideoWeb/VideoWeb.Services.Generator/video-api.nswag]: reverted to point to dev

**Comment by RocheL** [comment]
> This could easily be resolved with a guard (which exists but needs tweaking).
> Also the participants should not be able to see closed conferences (beyond 30 mins) in the list. These additional properties are not needed

Updated the ConferenceController to use the HasPassed check, added a guard for the participant-waiting -room.

File: VideoWeb/VideoWeb.Services.Generator/video-api.nswag (hunk 1 above)
Comment (npoxon) [reply VideoWeb/VideoWeb.Services.Generator/video-api.nswag]: cheers

**Pull request merged**